src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasGoogleCloudService.kt

```diff
@@ -90,60 +91,79 @@ class CosmasGoogleCloudService(private val freeBucketName: String,
         return BlobInfo.newBuilder(getBlobId(fileId, info)).build()
     }
 
+    private fun logging(funName: String, projectId: String, fileId: String = "", userId: String = "", other: Map<String, String> = mapOf(), body: () -> Unit) {
+        MDC.put("projectId", projectId)
```

Comment:
I think we should not put projectId, fileId or userId if they == "":

![image](https://user-images.githubusercontent.com/22072254/52640945-d1e13b00-2ee8-11e9-9e71-858b5da77370.png)

Reply:
fixed, thanks